a10_octavia/api/drivers/flavor_schema.py

```diff
@@ -183,6 +183,15 @@
         "device-name": {
             "type": "string",
             "description": "Specify vthunder device name that used for this loadbalancer"
+        },
+        "deployment": {
+            "type": "object",
+            "description": "Specify deployment strategy for the loadbalancer",
+            "properties": {
+                "dsr_type": {
```

Comment:
We better have description for "dsr_type", 
so with `openstack loadbalancer provider capability list a10` user can know what we support in the deployment flavor.

Reply:
Also added description for dsr_type